tests/test_cases/test_array_buses/Makefile

```diff
@@ -0,0 +1,3 @@
+include ../../designs/array_buses/Makefile
+
+MODULE = test_array_buses
```

Comment:
Since this is the only test that will use this design. I would move the design into the test directory. No need to keep it separate.

Reply:
Agreed, and done.